releasenotes/notes/qasm-lexer-bugfix-1779525b3738902c.yaml

```diff
@@ -0,0 +1,5 @@
+fixes:
+  - |
+    Fixes qasm lexer so the parser errors if version other than 2.0 is passed.
```

Comment:
We should be clear that only the QASM 2 tools will error when passed versions other than 2.0; the export capabilities for QASM 3 live in `qiskit.qasm3` - a separate module.